model/src/main/proto/question.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+package model;
+
+import "exploration.proto";
+
+option java_package = "org.oppia.app.model";
+option java_multiple_files = true;
+
+// Structure for a single question.
+message Question {
+  string question_id = 1;
+  State question_state = 2;
+  string language_code = 3;
+  int32 version = 4;
+  repeated string linked_skill_ids = 5;
+  int64 created_on_timestamp_ms = 6;
+  int64 updated_on_timestamp_ms = 7;
+}
+
+
```

Comment:
Nit: please remove extra blank newline (only one should terminate the file).

Reply:
Done